python.md

```diff
@@ -2,8 +2,54 @@
 layout: page
 title: Python
 ---
-TODO
+
+# General Tips
+
+* Try to follow [PEP8](https://www.python.org/dev/peps/pep-0008/), except for line length. Use a linter (like [pylint](http://www.pylint.org/)) to enforce them automatically in your code.
+* Use full names for variables where possible. For example, choose `job` over `j`, `something_longer` over `sl`, but not `this_variable_name_is_the_longest_name_evar_celery_man`.
+* Functions and classes should have at least minimal doc strings. A good guide to write them can be found in [PEP 0257](https://www.python.org/dev/peps/pep-0257/).
```

Comment:
They should have a docstring when it makes sense. If function name is self explanatory there is need for repeating that in docstring.